src/mnemonics/electrum-words.cpp

```diff
@@ -428,6 +431,13 @@ namespace crypto
       }
 
       words += words_store[create_checksum_index(words_store, language)];
+      std::time_t t = std::time(0);
```

Comment:
Look at epee/include/misc_os_dependent::get_gmt_time() and rpc_command_executor::get_date_time() for doing this.

You can subtract 1 days worth of seconds from here so you don't need to do the subtraction later and don't have to change the date range checks in wallet2.cpp